drape_frontend/gui/gui_text.cpp

```diff
@@ -240,6 +232,14 @@ void StaticLabel::CacheStaticText(std::string const & text, char const * delim,
 
   result.m_state.SetColorTexture(color.GetTexture());
   result.m_state.SetMaskTexture(buffers[0][0].GetTexture());
+
+  dp::TGlyphs glyphs;
+  for (auto const & line : shapedLines)
+    for (auto const & glyph : line.m_glyphs)
+      glyphs.emplace_back(glyph.m_key);
+
+  base::SortUnique(glyphs);
+  return glyphs;
```

Comment:
Still consider ```(dp::TGlyphs & glyphs)``` here and below.

Reply:
I'll make performance tests later, now you can check that assembly is bigger with your proposal compared to NRVO: https://editor.mergely.com/YPWIXRBm
https://godbolt.org/z/3dP9qao5q